canonical-web-and-design/maas-ui
# Set publicPath to '/MAAS/r/' in production.

## Done
* Set publicPath to '/MAAS/r/' in production.

## QA
* Build the app in production and ensure loading the app from any route works, e.g. `/MAAS/r/machine/<machineid>` or `/MAAS/l/device/<deviceid>`
